lambda/logserver/logserver.go

```diff
@@ -124,6 +124,15 @@ func (ls *LogServer) handler(res http.ResponseWriter, req *http.Request) {
 
 	for _, event := range logEvents {
 		switch event.Type {
+		case "extension":
+			record := event.Record.(string)
```

Comment:
line number 185-192 is duplicate with this code line, we can reutilize like if case extension OR function this execute line of code